org.sailfishos.usermanager.xml

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="org.sailfishos.usermanager">
+    <method name="users">
+        <arg direction="out" type="a(ssi)" name="users"/>
+        <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QList&lt;SailfishUserManagerEntry&gt;"/>
+    </method>
+    <method name="addUser">
```

Comment:
These don't have return values but would it make sense if they had? At least I see some benefit addUser would return uid or username (or user entry struct).

Reply:
Definitely needs to return uid, since username is provided as a parameter. Or both if the implementation is supposed to generate the username if you pass in an empty one.